app/models/miq_schedule.rb

```diff
@@ -217,6 +217,12 @@ def action_check_compliance(obj, _at)
     _log.info("Action [#{name}] has been run for target type: [#{obj.class}] with name: [#{obj.name}]")
   end
 
+  def action_automation_request(_klass, _at)
+    parameters = filter[:parameters]
+    user = User.find_by_userid(userid) || User.find_by_userid("admin")
```

Comment:
Why are we defaulting to admin if the userid isn't found?  I don't believe we want to elevate the user to admin here.  Shouldn't the schedule have a userid?  We have a default_value_for userid to "system".  How can it not have a userid?